<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>player_edit</class>
 <widget class="QWidget" name="player_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QToolBox" name="toolBox_2">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_properties">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>752</width>
        <height>395</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Properties</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="2">
          <widget class="QComboBox" name="players_tab_list_combo">
           <item>
            <property name="text">
             <string>1</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>2</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>4</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>5</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Player:</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Hit Points:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="players_tab_hp">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>32</number>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="players_tab_name">
           <property name="maxLength">
            <number>30</number>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QSpinBox" name="players_tab_maxshots">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Has Shield:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Max Shots:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="players_tab_hasshield">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_34">
           <property name="text">
            <string>Touch Damage Reductor::</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_35">
           <property name="text">
            <string>Jump Gravity:</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_36">
           <property name="text">
            <string>Jump Speed:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QDoubleSpinBox" name="players_tab_fallgravity">
           <property name="singleStep">
            <double>1.000000000000000</double>
           </property>
           <property name="value">
            <double>0.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QDoubleSpinBox" name="players_tab_jumpspeed">
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>9.000000000000000</double>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_37">
           <property name="text">
            <string>Move Speed:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QDoubleSpinBox" name="players_tab_jumpgravity">
           <property name="singleStep">
            <double>0.010000000000000</double>
           </property>
           <property name="value">
            <double>0.120000000000000</double>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Can Slide:</string>
           </property>
          </widget>
         </item>
         <item row="9" column="2">
          <widget class="QCheckBox" name="can_slide_checkbox">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Charged Shot:</string>
           </property>
          </widget>
         </item>
         <item row="10" column="2">
          <widget class="QComboBox" name="chargedshot_combo"/>
         </item>
         <item row="8" column="2">
          <widget class="QDoubleSpinBox" name="players_tab_movespeed">
           <property name="decimals">
            <number>1</number>
           </property>
           <property name="minimum">
            <double>0.500000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_graphics">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>735</width>
        <height>326</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Graphics</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="2" column="0">
            <widget class="QLabel" name="label_39">
             <property name="text">
              <string>Sprite Size:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_47">
             <property name="text">
              <string>Key Colors:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_41">
               <property name="text">
                <string>Width:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="player_sprite_w">
               <property name="value">
                <number>29</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_42">
               <property name="text">
                <string>Height:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="player_sprite_h">
               <property name="value">
                <number>32</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>Weapon Colors:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QPushButton" name="key1_picker">
               <property name="text">
                <string>Key 1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="ColorKeyIndicator1">
               <property name="minimumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="key2_picker">
               <property name="text">
                <string>Key 2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="ColorKeyIndicator2">
               <property name="minimumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="key3_picker">
               <property name="text">
                <string>Key 3</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="ColorKeyIndicator3">
               <property name="minimumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_38">
             <property name="text">
              <string>Graphics:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="color1_picker">
               <property name="text">
                <string>Color 1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="ColorValueIndicator1">
               <property name="minimumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="color2_picker">
               <property name="text">
                <string>Color 2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="ColorValueIndicator2">
               <property name="minimumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="color3_picker">
               <property name="text">
                <string>Color 3</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="ColorValueIndicator3">
               <property name="minimumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="2">
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="margin">
              <number>4</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_12">
               <item>
                <widget class="QLabel" name="label_43">
                 <property name="text">
                  <string>X:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="player_hitarea_x">
                 <property name="value">
                  <number>4</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_44">
                 <property name="text">
                  <string>Y:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="player_hitarea_y">
                 <property name="value">
                  <number>8</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_13">
               <property name="margin">
                <number>4</number>
               </property>
               <item>
                <widget class="QLabel" name="label_45">
                 <property name="text">
                  <string>W:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="player_hitarea_w">
                 <property name="value">
                  <number>21</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_46">
                 <property name="text">
                  <string>H:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="player_hitarea_h">
                 <property name="value">
                  <number>24</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_40">
             <property name="text">
              <string>Hit Area Size:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QComboBox" name="weaponlist_combo"/>
           </item>
           <item row="1" column="2">
            <widget class="QComboBox" name="player_graphics_combo"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Player:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="players_tab_list_combo_2">
             <item>
              <property name="text">
               <string>1</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>2</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>3</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>4</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>5</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QScrollArea" name="scrollArea_4">
           <property name="widgetResizable">
            <bool>false</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_4">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>489</width>
              <height>199</height>
             </rect>
            </property>
            <widget class="player_preview_area" name="player_preview_widget" native="true">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>471</width>
               <height>181</height>
              </rect>
             </property>
            </widget>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>player_preview_area</class>
   <extends>QWidget</extends>
   <header>player_preview_area.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
